packages/@react-spectrum/tabs/src/Tabs.tsx

```diff
@@ -0,0 +1,215 @@
+/*
+ * Copyright 2020 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import {classNames, useStyleProps} from '@react-spectrum/utils';
+import {CollectionChildren, CollectionElement, DOMProps, Node, Orientation, StyleProps} from '@react-types/shared';
+import {FocusRing} from '@react-aria/focus';
+import {ListState, useSingleSelectListState} from '@react-stately/list';
+import {mergeProps} from '@react-aria/utils';
+import {Picker} from '@react-spectrum/picker';
+import React, {Key, useEffect, useRef, useState} from 'react';
+import {SpectrumTabsProps} from '@react-types/tabs';
+import styles from '@adobe/spectrum-css-temp/components/tabs/vars.css';
+import tabsStyles from './tabs.css';
+import {useHover} from '@react-aria/interactions';
+import {useProviderProps} from '@react-spectrum/provider';
+import {useTab, useTabs} from '@react-aria/tabs';
+
+export function Tabs<T extends object>(props: SpectrumTabsProps<T>) {
+  props = useProviderProps(props);
+  let {
+    orientation = 'horizontal' as Orientation,
+    children,
+    onSelectionChange,
+    isDisabled,
+    isQuiet,
+    density,
+    defaultSelectedItem,
```

Comment:
I think you won't need this `defaultSelectedItem` prop since the `SingleSelection` type which the Tabs props will extend from provides the `selectedKey` and `defaultSelectedKey`. For a uncontrolled Tabs component it would look like: 
```
<Tabs defaultSelectedKey="val2">
...
</Tabs>
```
and a controlled Tabs component would looks like:
```
<Tabs selectedKey="files">
...
</Tabs>
```
You can see the Tabs api spec here: https://github.com/adobe/react-spectrum/issues/784#issuecomment-661585975. I'll do a more complete review soon, but just wanted to point some of the things I noticed from a quick glance. The stories look great btw :D, great progress